ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/vlan-interfaces.md

```diff
@@ -215,6 +218,9 @@
 
             # IPv4_address/Mask
           - <str>
+        ip_verify:
+          unicast_source:
+            reachable_via: <str; "any" | "rx">
```

Comment:
On the device where I tested, I could not find any other commands through this cli. I would consider collapsing to a single key: `ip_verify_unicast_source_reachable_via: <str; "any | "rx">`
@carlbuchmann thoughts?

Reply:
Please collapse to a single key `ip_verify_unicast_source_reachable_via` thanks.

Reply:
Fixed!